### PR TITLE
[Merged by Bors] - Cleanup tortoise beacon code after merging

### PR DESCRIPTION
**Pull request by nkryuchkov**
## Motivation

There are a few issues in merged https://github.com/spacemeshos/go-spacemesh/pull/2267. This PR fixes them.

## Changes

- Remove logs with sensitive information in `VRFSigner`
- Note non-resolved changes requested in code reviews of https://github.com/spacemeshos/go-spacemesh/pull/2267 as TODO's
- Revert p2p patch which is not needed
- Decrease tortoise beacon unit tests duration

## Test Plan

- Already existing unit and system tests

## DevOps Notes

- [x] This PR does not require configuration changes (e.g., environment variables, GitHub secrets, VM resources)
- [x] This PR does not affect public APIs
- [x] This PR does not rely on a new version of external services (PoET, elasticsearch, etc.)
- [x] This PR does not make changes to log messages (which monitoring infrastructure may rely on)

### CODE REVIEW

**Comment by nkryuchkov** [comment]
bors merge

File: tortoisebeacon/handlers.go [hunk 1]
```diff
@@ -75,7 +75,8 @@ func (tb *TortoiseBeacon) handleProposalMessage(ctx context.Context, m ProposalM
 		tb.Log.With().Warning("Received malformed proposal message: VRF is not verified",
 			log.String("sender", m.MinerID.String()),
 			log.String("sender_short", m.MinerID.ShortString()))
-		return nil
+
+		return ErrMalformedProposal
```
Comment (antonlerner) [inline tortoisebeacon/handlers.go:80]: this introduces another edge case, document that maybe additional test is needed for this one

**Review by antonlerner — APPROVED** [review]

**Comment by nkryuchkov** [comment]
bors merge

**Comment by nkryuchkov** [comment]
bors merge-

**Comment by nkryuchkov** [comment]
bors merge

**Comment by nkryuchkov** [comment]
bors merge

**Comment by nkryuchkov** [comment]
bors merge
